package.json

```diff
@@ -25,6 +25,8 @@
   "dependencies": {
     "firebase": "^4.9.1",
     "vue": "^2.5.2",
+    "vue-full-calendar": "^2.5.2",
+    "vue-fullcalendar": "^1.0.9",
```

Comment:
why are there two new dependencies being added to this file. shouldn't we have only one fullCalendar

Reply:
vue-fullcalendar is the one that I used earlier (the very basic one that doesn't require jquery)

Reply:
only 1.0.9 is needed